src/services/tx/txSender.ts

```diff
@@ -57,7 +57,19 @@ export const createTx = async (txParams: SafeTransactionDataPartial, nonce?: num
   // Set the recommended nonce and safeTxGas if not provided
   if (nonce === undefined) {
     const chainId = await safeSDK.getChainId()
-    const estimation = await estimateSafeTxGas(String(chainId), safeSDK.getAddress(), txParams)
+    const safeAddress = await safeSDK.getAddress()
+    let estimation: SafeTransactionEstimation
+    try {
+      estimation = await estimateSafeTxGas(String(chainId), safeSDK.getAddress(), txParams)
+    } catch (e) {
+      try {
+        const minTxParams = { ...txParams, data: '0x', to: safeAddress }
+        estimation = await estimateSafeTxGas(String(chainId), safeSDK.getAddress(), minTxParams)
+      } catch (e) {
+        console.error('Error estimating SafeTx gas', e)
```

Comment:
`console.error` -> `logError(ErrorCodes.xxx)`

Reply:
We have 3 potentially fitting error descs:
```
_612 = '612: Error estimating gas'
...
_616 = '616: Failed to retrieve recommended nonce',
_617 = '617: Error fetching safeTxGas',
```

I will add 617

Reply:
IMHO 616 fits better.